recipe/install_nvcc.sh

```diff
@@ -65,8 +65,13 @@ fi
 
 if [[ -z "\$(\${CUDA_HOME}/bin/nvcc --version | grep "Cuda compilation tools, release ${PKG_VERSION}")" ]]
 then
-  echo "Version of installed CUDA didn't match package"
-  return 1
+  if [[ ! -z "\${CONDA_BUILD+x}" ]]
```

Comment:
According to [the docs]( https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#environment-variables-set-during-the-build-process ), this is always set to `1` when it is set. So we could just compare to that value

The current statement would evaluate to `true` if it is setting to anything (including "" and "0"). Not sure that anything actually uses those other values, but we may still want those to evaluate to `false` for good measure

Reply:
Thanks John, updated with a strict comparison